package.json

```diff
@@ -46,6 +46,7 @@
     "commander": "2.9.0",
     "event-stream": "3.3.4",
     "express": "4.15.2",
+    "express-handlebars": "^3.0.0",
```

Comment:
Yeah, so a couple of things:

1. We don't compile handlebars in our build step. We just bundle them together. The browser compiles them when we need them. We can't compile those templates because we don't know the data yet.
2. We could do this for this file, but that would require rerunning the build whenever you change a config or add a theme or anything like that, which I would be very strongly against.